app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -26,6 +31,25 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['$http', '$scope',
     miqService.sparkleOff();
   }
 
+  $scope.storageManagerChanged = function (id) {
+    miqService.sparkleOn();
+    
+    $http.get('/cloud_volume/storage_manager_form_fields/' + id)
```

Comment:
I've seen [here](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/assets/javascripts/controllers/cloud_subnet/cloud_subnet_form_controller.js#L65-L71) that multiple requests are made to the backend, but decided to keep it simple for now.

Is it mandatory to request all field values separately or is it ok to have one function that will provide only the set of those choices that are applicable to the actual provider type?